is_core/static/is_core/css/app.css

```diff
@@ -7295,7 +7295,7 @@ form .errorlist {
     font-weight: bold; }
 
 .navbar-brand {
-  background: url("../img/mallpay/mallpay-logo.png") no-repeat center;
+  background: url("../images/mallpay/mallpay-logo.png") no-repeat center;
```

Comment:
Btw fakt chceme v IS Core logo MALL Pay?

Reply:
to tam vůbec být nemá. Pryč s tím